Makefile

```diff
@@ -306,11 +309,11 @@ $(PKG): release-only
 	rm -rf $(PKGDIR)
 	rm -rf out/deps out/Release
 	$(PYTHON) ./configure --download=all --with-intl=small-icu \
-		--without-snapshot --dest-cpu=ia32 --tag=$(TAG)
```

Comment:
@misterdjules : do you build releases with snapshots or without? I'd argue that the default should be in line with how releases are built. 

But if that doesn't work, we could add a configure-for-ci parameter to configure.py.